file-formats/uist/type/zif_aff_uist_v1.intf.abap

```diff
@@ -32,35 +46,15 @@ INTERFACE zif_aff_uist_v1
       "! <p class="shorttext">Header</p>
       "! Header
       "! $required
-      header          TYPE zif_aff_types_v1=>ty_header_100,
+      header          TYPE zif_aff_types_v1=>ty_header_60_cloud,
```

Comment:
It is my mistake, I generate a files from Interface in ABAP and there ty_header_100_cloud is not available yet. Temporary I set to _60_ to avoid abap error and forgot to change is in a file.